### PR TITLE
Added copy for success messages, added logic in success router to asc…

### PR DESCRIPTION
**Pull request by joefitter**
…ertain which message to display

### CODE REVIEW

File: pages/common/routers/success.js [hunk 1]
```diff
@@ -1,8 +1,24 @@
+const { merge, get } = require('lodash');
 const { Router } = require('express');
+const successContent = require('../content/success-messages');
 
-module.exports = ({ model = 'model' } = {}) => {
+module.exports = ({
+  model = 'model',
+  licence = 'pel',
+  type = 'amendment'
```
Comment (lennym) [inline pages/common/routers/success.js]: I'm not sure having default values here is appropriate. What's the reasoning behind them?

Comment (joefitter) [reply pages/common/routers/success.js]: most changes are pel amendments

File: pages/common/routers/success.js [hunk 2]
```diff
@@ -1,8 +1,24 @@
...
+} = {}) => {
   const app = Router();
 
+  app.use((req, res, next) => {
+    if (req.model && req.model.status) {
+      type = req.model.status === 'active' ? 'amendment' : 'application';
+    }
+    const success = get(successContent, `${licence}.${type}.resubmitted`);
```
Comment (lennym) [inline pages/common/routers/success.js]: Hard-coded `resubmitted` strikes me as odd.

Comment (joefitter) [reply pages/common/routers/success.js]: yeah - it could be passed in, but it is resubmitted 100% of the time

Comment (joefitter) [reply pages/common/routers/success.js]: the logic for displaying task success messages lives elsewhere

Comment (joefitter) [reply pages/common/routers/success.js]: also resubmitted === submitted

File: pages/common/routers/success.js (hunk 1 above)
Comment (joefitter) [reply pages/common/routers/success.js]: but yeah, I'll pass them in

**Pull request merged**